docker-entrypoint.sh

```diff
@@ -4,6 +4,19 @@ cd rems
 
 [ -z "$COMMANDS" ] && COMMANDS="run"
 
+certfile=$(ls /rems/certs)
```

Comment:
should this be documented somewhere?

Reply:
Is this sufficient? https://github.com/CSCfi/rems/blob/findata-cert/docs/docker.md